src/helpers/fallbackauthdialog.ts

```diff
@@ -4,29 +4,61 @@
  * This file shows how to use MSAL.js to get an access token to Microsoft Graph an pass it to the task pane.
  */
 
-/* global console, localStorage, Office */
+/* global console, localStorage, location, Office, window */
 
-import { Configuration, PublicClientApplication, RedirectRequest } from "@azure/msal-browser";
+import { Configuration, LogLevel, PublicClientApplication, RedirectRequest } from "@azure/msal-browser";
+import { callGetUserData } from "./middle-tier-calls";
+import { showMessage } from "./message-helper";
+
+const clientId = "{application GUID here}"; //This is your client ID
+const accessScope = `api://${window.location.host}/${clientId}/access_as_user`;
+const loginRequest: RedirectRequest = {
+  scopes: [accessScope],
+  extraScopesToConsent: ["user.read"],
+};
 
 const msalConfig: Configuration = {
   auth: {
-    clientId: "{application GUID here}", //This is your client ID
+    clientId: clientId,
     authority: "https://login.microsoftonline.com/common",
-    redirectUri: "https://localhost:{PORT}/fallbackauthdialog.html",
+    redirectUri: "https://localhost:{PORT}/fallbackauthdialog.html", // Update config script to enable `https://${window.location.host}/fallbackauthdialog.html`,
     navigateToLoginRequestUrl: false,
```

Comment:
I also had to include `response_type: "access_token"`, but if you are getting the correct access token to the node server, then perhaps it is not needed in all cases.

Reply:
The typescript type says that's not a valid property so I'm leaving it off.

Reply:
sounds good. Maybe something changed in MSAL as you are using a later version than the sample I put together.